webpack.config.js

```diff
@@ -9,19 +9,42 @@
  */
 
 var webpack = require('webpack');
-var ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+var filename;
+var vendorName;
+var plugins = [];
+
+if(process.env.BUNDLE_DEPS) {
+  filename = 'keyword_in_context.all.js';
+} else {
+  filename = 'keyword_in_context.js';
+  vendorName = 'keyword_in_context.deps.js';
+  plugins.push(
+    new webpack.optimize.CommonsChunkPlugin('vendor', vendorName)
+  );
+}
+
+if(process.env.MINIFY) {
+  plugins.push(
+    new webpack.optimize.UglifyJsPlugin({ compress: { warnings: false } })
+  );
+}
 
 module.exports = {
-  context: __dirname + '/src',
+  context: __dirname,
   entry: {
-    app: './index.jsx',
-    vendor: ['react', 'd3', 'lodash']
+    main: './index.js',
+    vendor: ['react', 'react-dom', 'd3', 'lodash']
+  },
+  resolve: {
```

Comment:
btw @vlandham if you are trying to adapt this to concordance plots. do not forget this line/object, its the little innocuous looking line that not having sent me round in circles for ages.